plugins/chat/app/serializers/chat_channel_serializer.rb

```diff
@@ -110,6 +110,7 @@ def current_user_membership
   def meta
     {
       message_bus_last_ids: {
+        channel_message_bus_last_id: MessageBus.last_id("/chat/#{object.id}"),
```

Comment:
This was causing some issues in tests as we were getting the id only once we had loaded messages